infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources/kubernetes-external-secrets.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kubernetes-external-secrets
+  namespace: kubernetes-external-secrets
+  labels:
+    app: kubernetes-external-secrets
+spec:
+  selector:
+    app: kubernetes-external-secrets
+  ports:
+    - protocol: TCP
+      port: 3001
+      name: prometheus
+      targetPort: prometheus
```

Comment:
@spiffxp are you planning to have a monitoring stack in this cluster ?

Reply:
Regardless of whether there is a prometheus in this cluster, something should eventually scrape this. We've got metrics exposed by greenhouse and boskos in k8s-infra-prow-build.  Let me double-check whether I have this exposed similarly

Reply:
I add a `google_compute_address` in terraform and made this of type `LoadBalancerIP` like the other two prow services that currently get scraped by monitoring.k8s.io